src/features/user/UserProfile/styles/EditProfileModal.module.scss

```diff
@@ -48,14 +48,13 @@
 
 .editProfileText {
     font-size: 22px;
-    margin-left: 10px;
+    margin-left: 40px;
```

Comment:
Why is this extra space added ? is that intentional ? @captainamiedi 

<img width="272" alt="Screenshot 2020-11-19 at 11 25 19 AM" src="https://user-images.githubusercontent.com/25605193/99627261-efe2b480-2a59-11eb-8578-ea04243facb9.png">

Reply:
Yes it is. i thought the back arrow and edit profile is to close. but if that's not what the design state, i can reverse it